test/spec.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../web-component-tester/browser.js"></script>
+
+    <link rel="import" href="../juicy-jsoneditor.html">
+</head>
+
+<body>
+    <test-fixture id="element">
+        <template>
+            <juicy-jsoneditor></juicy-jsoneditor>
+        </template>
+    </test-fixture>
+</body>
+<script>
+    var juicyElement;
+    describe('juicy-jsoneditor', function() {
+
+        beforeEach(function() {
+            juicyElement = fixture('element');
+        });
+
+        it('should be upgraded element', function() {
+            expect(juicyElement).to.be.an.instanceOf(HTMLElement);
+            expect(juicyElement).to.be.not.an.instanceOf(HTMLUnknownElement);
```

Comment:
FYI, AFAIK this `HTMLUnknownElement` check only works in Safari See: https://jsbin.com/bokijurija/edit?html,js,console

I'm not sure this test will cover syntax errors.

Reply:
ha! Great catch! thanks

What about https://jsbin.com/liqexan/1/edit?js,console
```js
expect(juicyElement).to.be.an.instanceOf(HTMLElement);
expect(customElements.get('some-random-element')).to.be.defined;
expect(juicyElement).to.be.not.an.instanceOf(customElements.get('some-random-element'));
```

Reply:
It will cover errors in constructor

Reply:
It's certainly better but `expect(juicyElement).to.be.an.instanceOf(HTMLElement);` will always pass. 

`expect(customElements.get('some-random-element')).to.be.defined;`, perfect.

`customElements.get('some-random-element')` this can return undefined. Which breaks `instanceOf` I say `expect(customElements.get('some-random-element')).to.be.ok` is safer. Although it's looser than `.defined`. So I would have the test like this:

```js
expect(customElements.get('some-random-element')).to.be.ok;
expect(customElements.get('some-random-element')).to.be.defined;
```

Reply:
Sorry, I misunderstod `defined`. I thought it meant `a defined custom element`, not the opposite of `defined`. Ignore the comment above.